webui/src/Buttons/EditButton/FeedbackPanel.jsx

```diff
@@ -255,6 +255,41 @@ export function FeedbackEditor({ feedback, setValue, innerDelete, setSelectedSty
 	const feedbackSpec = (feedbacksContext[feedback.instance_id] || {})[feedback.type]
 	const options = feedbackSpec?.options ?? []
 
+	const [optionVisibility, setOptionVisibility] = useState({})
+
+	useEffect(() => {
+		const options = feedbackSpec?.options ?? []
+
+		console.log({ options })
+
+		for (const option of options) {
+			if (typeof option.isVisibleFn === 'string') {
+				option.isVisible = sandbox(option.isVisibleFn)
+			}
+		}
+	}, [feedbackSpec, feedback])
```

Comment:
I don't think this `feedback` reference is needed. It should be removed to avoid invalidations of the effect